worknenjoy/gitpay
# Fixing exhibition of paypal payment retry button

## Description

> Hotfix of retry button for failed paypal payments.

## Changes

- [x] Task list

## Status

> On review

## Related Issue

> #256 

## Impacted Area

> Main page

## Steps to test

- `git clone git@github.com:worknenjoy/gitpay.git`
- `cd gitpay/frontend`
- `npm install`
- `npm run dev`
- Go to http://localhost:8082/
- Create a task
- Add a bounty with Paypal
- Cancel the checkout
- You will see a pending payment with a retry button only for the owner of order
